### PR TITLE
Update create-external-data-source-transact-sql.md

### PR DESCRIPTION
**Pull request by joannapea**

### CODE REVIEW

File: docs/t-sql/statements/create-external-data-source-transact-sql.md [hunk 1]
```diff
@@ -2083,11 +2083,11 @@ Takes a shared lock on the `EXTERNAL DATA SOURCE` object.
 
 PolyBase supports proxy based authentication for most external data sources. Create a database scoped credential to create the proxy account.
 
-When you connect to the storage or data pool in a SQL Server 2019 Big Data Cluster, the user's credentials are passed through to the back-end system. Create logins in the data pool itself to enable pass through authentication.
+Shared Access Signature (SAS) keys are supported for authenticating to Azure Data Lake Store Gen 2 Storage Accounts. Customers that want to authenticate using a Shared Access Signature must create a database scoped credential where IDENTITY = "Shared Access Signature" and enter a SAS token as the secret. Customers that want to authenticate using a Storage Account Key must use a database scoped credential where IDENTITY = “Storage Account Key” and use a storage key as the secret.
```
Comment (WilliamDAssafMSFT) [inline docs/t-sql/statements/create-external-data-source-transact-sql.md:2086]:
```suggestion
Shared Access Signature (SAS) keys are supported for authenticating to Azure Data Lake Store Gen 2 Storage Accounts. Customers that want to authenticate using a Shared Access Signature must create a database scoped credential where `IDENTITY = "Shared Access Signature"` and enter a SAS token as the secret. 
```

File: docs/t-sql/statements/create-external-data-source-transact-sql.md [hunk 2]
```diff
@@ -2083,11 +2083,11 @@ Takes a shared lock on the `EXTERNAL DATA SOURCE` object.
...
 
-When you connect to the storage or data pool in a SQL Server 2019 Big Data Cluster, the user's credentials are passed through to the back-end system. Create logins in the data pool itself to enable pass through authentication.
+Shared Access Signature (SAS) keys are supported for authenticating to Azure Data Lake Store Gen 2 Storage Accounts. Customers that want to authenticate using a Shared Access Signature must create a database scoped credential where IDENTITY = "Shared Access Signature" and enter a SAS token as the secret. Customers that want to authenticate using a Storage Account Key must use a database scoped credential where IDENTITY = “Storage Account Key” and use a storage key as the secret. 
 
-An SAS token with type `HADOOP` is unsupported. It's only supported with type = `BLOB_STORAGE` when a storage account access key is used instead. Attempting to create an external data source with type `HADOOP` and a SAS credential fails with the following error:
+If a customer creates a database scoped credential where IDENTITY = ‘Shared Access Signature’ and uses a storage key value as the secret, they will get the following error:
```
Comment (WilliamDAssafMSFT) [inline docs/t-sql/statements/create-external-data-source-transact-sql.md:2088]:
```suggestion
If you create a database scoped credential where `IDENTITY = "Shared Access Signature"` and uses a storage key value as the secret, you will get the following error message: 
```

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]
A couple minor revisions @joannapea. Removed a repeated sentence. You can "commit suggestions" in a couple places to apply them.

**Comment by Court72** [comment]
@WilliamDAssafMSFT 

Thanks for reviewing the proposed changes. *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by joannapea** [comment]
@SnehaGunda  Would you mind helping since William is out?

**Comment by WilliamDAssafMSFT** [comment]
Looks good to me. #sign-off

**Pull request merged**